src/geocat/comp/interp_hybrid_to_pressure.py

```diff
@@ -86,26 +90,76 @@ def _vertical_remap(data, pressure):
 
         return func_interpolate(new_levels, pressure, data, axis=interp_axis)
 
-    # Apply vertical interpolation
-    # Apply Dask parallelization with xarray.apply_ufunc
-    output = xr.apply_ufunc(
+    ###############################################################################
+    # Workaround
+    #
+    # For the issue with metpy's xarray interface:
+    #
+    # `metpy.interpolate.interpolate_1d` had "no implementation found for
+    # 'numpy.apply_along_axis'" issue for cases where the input is
+    # xarray.Dataarray and has more than 3 dimensions (e.g. 4th dim of `time`).
+
+    # Use dask.array.core.map_blocks instead of xarray.apply_ufunc and
+    # auto-chunk input arrays to ensure using only Numpy interface of
+    # `metpy.interpolate.interpolate_1d`.
+
+    # # Apply vertical interpolation
+    # # Apply Dask parallelization with xarray.apply_ufunc
+    # output = xr.apply_ufunc(
+    #     _vertical_remap,
+    #     data,
+    #     pressure,
+    #     exclude_dims=set((lev_dim,)),  # Set dimensions allowed to change size
+    #     input_core_dims=[[lev_dim], [lev_dim]],  # Set core dimensions
+    #     output_core_dims=[["plev"]],  # Specify output dimensions
+    #     vectorize=True,  # loop over non-core dims
+    #     dask="parallelized",  # Dask parallelization
+    #     output_dtypes=[data.dtype],
+    #     dask_gufunc_kwargs={"output_sizes": {
+    #         "plev": len(new_levels)
+    #     }},
+    # )
+
+    # If an unchunked Xarray input is given, chunk it just with its dims
+    if (data.chunks is None):
```

Comment:
```suggestion
    if data.chunks is None:
```
redundant parenthesis